templates/advantage/index.html

```diff
@@ -226,318 +140,38 @@ <h2>Your enterprise subscriptions</h2>
   <div class="row">
     <div class="col-6 u-sv3">
       <h2>Your free personal subscription</h2>
-      <p>By using Ubuntu Avantage you agree to our <a href="/legal/ubuntu-advantage-service-description">terms and conditions</a>.</p>
-      <aside class="p-accordion" role="tablist" aria-multiselect="true" style="border: 1px solid #cdcdcd;">
-        <ul class="p-accordion__list" style="margin-bottom: 0;">
-          <li class="p-accordion__group" style="background: white;">
-            <button type="button" class="p-accordion__tab" id="tab1" role="tab" aria-controls="#tab1-section" aria-expanded="false" onclick="dataLayer.push({
-              'event' : 'GAEvent',
-              'eventCategory' : 'Advantage',
-              'eventAction' : 'Show instructions',
-              'eventLabel' : 'Personal UA token',
-              'eventValue' : undefined
-            });">Show instructions</button>
-            <section class="p-accordion__panel" id="tab1-section" role="tabpanel" aria-hidden="true" aria-labelledby="tab1-section" style="padding: 0 1rem;">
-              <p>To attach a machine:</p>
-              <div class="p-code-copyable">
-                <input class="p-code-copyable__input" value="sudo ua attach {{ personal_account.free_token }}" readonly="readonly">
-                <button class="p-code-copyable__action" onclick="dataLayer.push({
-                  'event' : 'GAEvent',
-                  'eventCategory' : 'Advantage',
-                  'eventAction' : 'Copy UA attach command',
-                  'eventLabel' : 'Personal UA token',
-                  'eventValue' : undefined
-                });">Copy to clipboard</button>
-              </div>
-
-              <p>To detach:</p>
-              <div class="p-code-copyable">
-                <input class="p-code-copyable__input" value="sudo ua detach" readonly="readonly">
-                <button class="p-code-copyable__action" onclick="dataLayer.push({
-                  'event' : 'GAEvent',
-                  'eventCategory' : 'Advantage',
-                  'eventAction' : 'Copy UA detach command',
-                  'eventLabel' : 'Personal UA token',
-                  'eventValue' : undefined
-                });">Copy to clipboard</button>
-              </div>
-            </section>
-          </li>
-        </ul>
-      </aside>
+        <p>To attach a machine:</p>
+        <pre style="border: 0; padding: 0;"><code>sudo ua attach {{ personal_account.free_token }}</code></pre>
+        <p>For further options <a href="">visit the ua manpage</a>.</p>
```

Comment:
“Further” suggests that nothing more will be mentioned here. So move this line after the instructions for individual services.

Reply:
Done